10LessonTen/Checkpoint2/css/style.css

```diff
@@ -0,0 +1,129 @@
+body{
+  font-family: walter turncoat;
+}
+
+
+a{
+  text-decoration: none;
+
+}
+
+ul{
+  list-style-type: none;
+  display:flex;
+
+  }
+
+nav{
+  display:flex;
+  justify-content: space-between;
+}
+
+nav p{
+  font-size: 20px;
+  font-weight: bold;
+
+}
+
+
+li{
+  margin-right: 30px;
+  font-size: 20px;
+  color:black;
+}
+
+header{
+  background-image:url(https://scontent.xx.fbcdn.net/v/t1.0-9/13891817_289499371415232_230777946967839666_n.jpg?oh=1f42c8042d546591e09a5dcd4ce3918e&oe=58102E99);
+    height:500px;
+    display:flex;
+    flex-direction:column;
+    align-items:center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-attachment: fixed;
+
+ }
+header h1, header p{
+    font-family:walter turncoat;
+  }
+
+header h1{
+  font-size:100px;
+  color:red;
+  margin-bottom:30px;
+  font-weight: bold;
+  }
+
+header p{
+  color:white;
+  font-size: 30px;
+  font-weight: bolder;
+
+}
+
+#first{
+  display:flex;
+  margin-bottom:70px;
+  margin-top:20px;
+}
+
+#second{
+ display:flex;
+ margin-bottom: 40px;
+}
```

Comment:
I'd consider adding some margin on the sides of this div; most web-users don't like when text runs _right_ up to the edge of the page (in particular, `margin-left` in this case).